src/components/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+
+export const Countdown = () => {
+  const [timeLeft, setTimeLeft] = useState({
+    days: 0,
+    hours: 0,
+    minutes: 0,
+    seconds: 0
+  });
+
+  useEffect(() => {
+    setInterval(() => {
+      const countdownDate = new Date('2024-11-20');
+      const now = new Date();
+
+      const timeDifference = countdownDate.getTime() - now.getTime();
+
+      const days = Math.floor(timeDifference / (1000 * 60 * 60 * 24));
+      const hours = Math.floor((timeDifference / (1000 * 60 * 60)) % 24);
+      const minutes = Math.floor((timeDifference / (1000 * 60)) % 60);
+      const seconds = Math.floor((timeDifference / 1000) % 60);
+
+      setTimeLeft({ days, hours, minutes, seconds });
+    }, 1000);
+  }, []);
+
+  const formatNumber = (number) => {
+    return number < 10 ? `0${number}` : number;
+  };
+
+  return (
+    <article className="flex flex-wrap justify-center items-center gap-24">
```

Comment:
```suggestion
    <>
```

Recomiendo envolver Countdown en un fragmento de React para aumentar su reutilizabilidad. Así, SimpleCard puede controlar el estilo y contenido, haciendo Countdown más adaptable.